parameter/DefaultElementLibrary.h

```diff
@@ -71,15 +78,29 @@ class CDefaultElementLibrary: public CElementLibrary
       */
     CElement* createElement(const CXmlElement& xmlElement) const;
 
+    /**
+     * Application logger getter
+     *
+     * FIXME: Avoid the use of this getter by reworking plugin builder declaration.
+     *
+     * @return the logger provided by the client
+     */
+    core::log::ILogger& getLogger();
+
 private:
     bool _bEnableDefaultMechanism;
     CDefaultElementBuilder _DefaultElementBuilder;
+
+    /** Application Logger we need to provide to the plugin */
+    core::log::ILogger& _Logger;
```

Comment:
Why? Current coding style uses a capital letter after the underscore like 
~~~c++
CDefaultElementBuilder _DefaultElementBuilder;
~~~
or this is a mistake too ?